definitions/pretome.yml

```diff
@@ -0,0 +1,90 @@
+---
+  site: pretome
+  name: PreToMe
+  description: "a general tracker"
+  language: en-us
+  links:
+    - https://pretome.info/
+
+  caps:
+    categories:
+      22: PC              # Applications
+      27: Books           # Ebooks
+      4: PC/Games         # Games
+      31: Other           # Miscellaneous
+      19: Movies          # Movies
+      6: Audio            # Music
+      7: TV               # TV
+      51: XXX             # XXX
+
+    modes:
+      search: [q]
+      tv-search: [q, season, ep]
+
+  settings:
+    - name: pin
+      type: text
+      label: Pin
+    - name: username
+      type: text
+      label: Username
+    - name: password
+      type: password
+      label: Password
+
+  login:
+    path: /takelogin.php
+    method: post
+    form: form
+    inputs:
+      returnto: "/"
+      login_pin: "{{ .Config.pin }}"
+      username: "{{ .Config.username }}"
+      password: "{{ .Config.password }}"
+      login: "Login"
+    error:
+      - selector: font[color="red"]:has(h1)
+        message:
+          selector: font[color="red"]
+    test:
+      path: /my.php
+  ratio:
+    text: "8"

```

Comment:
An encoding issue. It's supposed to be a infinity symbol (it's a ratioless tracker).